robottelo.properties.sample

```diff
@@ -58,13 +58,24 @@ ssh_key=
 # Binary location for selected wedriver
 # webdriver_binary=/usr/bin/firefox
 # webdriver_binary=/usr/bin/chromedriver
-# cdn=true
-# Run one datapoint or multiple datapoints for tests
-# run_one_datapoint=false
+
+# webdriver_desired_capabilities accepts extra configuration to be passed
+# to saucelabs browser in order to configure the platform. You can use
+# the platform configurator (see link below) to get the information about
+# the environment you want to test.
+# https://wiki.saucelabs.com/display/DOCS/Platform+Configurator/
+# PS.: the base DesiredCapabilities dict will be get by the browser
+# specified by webdriver config. If you override browserName then that
+# browser will be used instead.
+#webdriver_desired_capabilities=browserName=firefox,platform=OS X 10.11,version=44.0,screenResolution=1024x768
```

Comment:
Could we also add an additional parameter ``"build": "jenkins"``? What I'd like to do then is, during an automated run, modify the value ``jenkins`` to match what we currently use for a job `label`. This will help us tell tests apart in Sauce Labs.